src/Log.hpp

```diff
@@ -12,8 +12,11 @@ class LoggerStream;
 class Logger
 {
 public:
-    Logger();
+    Logger(bool debug = false);
     LoggerStream createLogMessage(const std::string& file, const std::string& function, int line);
+    void setDebugMode(bool debug);
+    bool isEnabled();
+    bool _debug;
```

Comment:
why is this public?

Reply:
``setDebug()`` and ``isEnabled()`` are called from main. The can not be private within this context. However, ``_debug`` should be.

Reply:
I meant only _debug